userspace/libsinsp/fields_info.h

```diff
@@ -24,5 +24,5 @@ class sinsp;
 //
 // Printer functions
 //
-void list_fields(bool verbose, bool markdown, bool names_only=false);
+void list_fields(bool verbose, bool markdown=false);
```

Comment:
Is there a reason why names_only support was dropped?
Maybe i am just missing something :)

Reply:
This function is only used by oss sysdig, and oss sysdig doesn't actually have an option to only print field names. All the calls to list_fields in the sysdig repo omit the third argument.

Falco has its own function to print field info, and does support a names only argument.